### ISSUE TITLE
Issues with DMN Editor on macos

### ISSUE THREAD

**Issue by zellerman** [camunda/camunda-modeler#1778]
__Describe the Bug__

1. The modeler crashes when I try to change the id of the decision table.
2. The modeler gui doesn't refresh when I press the '+' button or remove from the context menu  on the dmn table row or columns, unless I switch to  "View DRD" and back.
3. While writing text to  the cells in the rows of the dmn table, after about 1 second the value resets unless I tab out the cell before the 1 second is elapsed.

Stack trace of 1.:
```
[/Users/zeller/Desktop/diagram_2.dmn] Cannot use 'in' operator to search for 'id' in undefined
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/7.7.js:8:187829
    at oldProperties (webpack:///node_modules/dmn-js-shared/lib/features/modeling/behavior/IdChangeBehavior.js:69:25)
    at shouldSkipUpdate (webpack:///node_modules/dmn-js-shared/lib/features/modeling/behavior/IdChangeBehavior.js:50:15)
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/1.1.js:1:50614
    at listener (webpack:///node_modules/diagram-js/lib/core/EventBus.js:365:33)
    at _invokeListener (webpack:///node_modules/diagram-js/lib/core/EventBus.js:350:23)
    at _invokeListeners (webpack:///node_modules/diagram-js/lib/core/EventBus.js:311:23)
    at fire (webpack:///node_modules/diagram-js/lib/command/CommandStack.js:346:28)
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/1.1.js:1:90220
    at fn (webpack:///node_modules/diagram-js/lib/command/CommandStack.js:367:4) [ error ]
[/Users/zeller/Desktop/diagram_2.dmn] Cannot use 'in' operator to search for 'id' in undefined
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/7.7.js:8:187829
    at oldProperties (webpack:///node_modules/dmn-js-shared/lib/features/modeling/behavior/IdChangeBehavior.js:69:25)
    at shouldSkipUpdate (webpack:///node_modules/dmn-js-shared/lib/features/modeling/behavior/IdChangeBehavior.js:50:15)
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/1.1.js:1:50614
    at listener (webpack:///node_modules/diagram-js/lib/core/EventBus.js:365:33)
    at _invokeListener (webpack:///node_modules/diagram-js/lib/core/EventBus.js:350:23)
    at _invokeListeners (webpack:///node_modules/diagram-js/lib/core/EventBus.js:311:23)
    at fire (webpack:///node_modules/diagram-js/lib/command/CommandStack.js:346:28)
    at file:///private/var/folders/x7/r3r9jthn3kg8_xqfv0z7rwdr0000gp/T/AppTranslocation/AE7E0F95-DEE5-437D-B441-F972F517C41F/d/Camunda%20Modeler.app/Contents/Resources/app.asar/public/1.1.js:1:90220
    at fn (webpack:///node_modules/diagram-js/lib/command/CommandStack.js:367:4) [ error ]
```
__Steps to Reproduce__

Follow the DMN tutorial, try setting the id of the table, edit the cells/columns.

__Expected Behavior__

1. Do not crash and update the id.
2. Refresh the gui
3. Do not overwrite the value of the cells while I'm typing.

__Environment__

- OS: Macos 10.15.3
- Camunda Modeler Version: 3.7.2

**Comment by pinussilvestrus** [comment]
Thanks for reporting! This seems to be a duplicate of https://github.com/camunda/camunda-modeler/issues/1769. Maybe you can try out the [4.0.0-alpha.1](https://blog.camunda.com/post/2020/03/camunda-modeler-4.0.0-alpha.1-released/) version of the application? The bug is not visible there anymore.

**Comment by zellerman** [comment]
Thanks, I can confirm that the issue is not present it this version. Are there any known issues with this alpha release I should be aware of?

**Comment by nikku** [comment]
Yes, cf. [changelog](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md#400-alpha0). 

TLDR: You should not use the alpha release in production, as it exports DMN 1.3 (not compatible with stable, released versions of the Camunda BPM platform).

**Comment by nikku** [comment]
Closing this issue as a duplicate.

**Comment by zellerman** [comment]
@nikku Thanks, in the meantime which editor version can I use to avoid the issue and use the model with the workflow engine?

**Comment by nikku** [comment]
Camunda Modeler 3.5 is a version that you can safely use.
